x-pack/lib/monitoring/internal_pipeline_source.rb

```diff
@@ -73,6 +73,14 @@ def populate_license_state(xpack_info)
             :log_level => :error,
             :log_message => "Monitoring is not available: #{xpack_info.license_type} is not a valid license for this feature."
         }
+      elsif !xpack_info.feature_enabled?(FEATURE) && is_serverless
+        {
+          :state => :error,
+          :log_level => :error,
+          :log_message => "Internal collection for monitoring is enabled in Logstash, but is not supported in the configured version of Elasticsearch.\n"\
+            "Please configure Elastic Agent to monitor Logstash. Documentation can be found at: \n"\
```

Comment:
I'd like to add emphasis that we're not sending monitoring

```suggestion
            "This instance of Logstash will NOT send its monitoring data to Elasticsearch. Please configure Elastic Agent to monitor Logstash. Documentation can be found at: \n"\
```